spec/requests/admin/justifications_spec.rb

```diff
@@ -65,9 +65,10 @@
     end
   end
 
-  describe '#destroy' do
+  describe 'DELETE #destroy' do
     it 'deletes a justification' do
-      expect { delete admin_justification_path(justifications.first) }.to change(Justification, :count).by(2)
+      justification = create(:justification)
+      expect { justification.destroy }.to change(Justification, :count).by(-1)
```

Comment:
@basmoura I've tried to `delete :destroy, id: justification` but I'm getting this error ArgumentError: unknown keyword: id. Do you know why?

Reply:
@davithss thats because Rails is expecting a set of params as this is a request, so you need to specify like `params: { id: your_id }`

Reply:
@basmoura Neither this way works! :/